packages/kbn-test-subj-selector/index.d.ts

```diff
@@ -18,4 +18,4 @@
  */
 
 // eslint-disable-next-line import/no-default-export
-export default function(selector: string): string;
+export default function kbnTestSubjSelector(selector: string): string;
```

Comment:
Named this to avoid https://github.com/typescript-eslint/typescript-eslint/issues/1078